docs/javascript/v2/features/simulcast.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Simulcast
+nav: 3.33
+---
+
+To turn on simulcast in your room, open the [templates](https://dashboard.100ms.live/templates) in the dashboard and enable it for roles there.
+
+
+## Responding in the app
```

Comment:
we should rename to section to "Setting Simulcast Layer" and just explain that(i.e. knowing available levels, and action to change).
We can add another section "Layer selection based on Video Element's Dimension" and tell that this is auto handled.